docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md

```diff
@@ -12,51 +12,191 @@ The syntax of `MATCH` is relatively more flexible compared with that of other qu
 MATCH <pattern> [<WHERE clause>] RETURN <output>
 ```
 
-## Patterns and WHERE clauses
+## The workflow of MATCH
 
-The `MATCH` patterns, with the help of curly brackets ("{}"), can do the same thing with `WHERE` clauses when making equality comparisons. You may find an example as follows.
+The `MATCH` statement relies on a **native index** to find a vertex as the origin of the search process. In other words, in a valid `MATCH` statement, **there must be an indexed property, or a specific VID.** For how to index a property, see [Create native index](../14.index-statements/1.create-native-index.md).
```

Comment:
The `MATCH` statement relies on a **native index**  or a **text-based index** to

Reply:
Not really... Only native index for now.